source/_integrations/unifi.markdown

```diff
@@ -112,13 +112,7 @@ Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfi
 
 ## Sensor
 
-These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.
-
-1. To enable the sensors, on the UniFi integration page, select **Configure**. 
-2. Go to page 3/3 and enable the sensors.
-
-Currently, it is not possible to get a global bandwidth sensor for a UDM router. The integration only supports per-device sensors.
-
+These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks. The integration only supports per-device sensors. Get entities reporting receiving and transmitting bandwidth per network client. To enable the bandwidth sensors, on the UniFi integration page, select **Configure**, go to page 3/3 and enable the bandwidth sensors.
```

Comment:
This is still in sensors rather than u Der bandwidth sensor

Reply:
I see. I misunderstood your previous comment. Let me try this again.

Reply:
Excellent! :)

Reply:
:) will you approve?

Reply:
Nearly there

Reply:
Thank you for bearing with me.